tests/Unit/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Test_TovStar.cpp

```diff
@@ -42,6 +42,32 @@ void verify_solution(const TovStar& solution, const std::array<double, 3>& x) {
   const auto domain = brick.create_domain();
   verify_grmhd_solution(solution, domain.blocks()[0], mesh, 1.e-7, 1.234,
                         1.e-4);
+
+  // Check that analytically computed dp/dx^i actually equals numerical
+  // derivative of pressure
```

Comment:
Looks good! I think you can just use `numerical_derivative` from TestHelpers.hpp here. Something like this should do it:

```cpp
  const auto dpdx_numerical = numerical_derivative(
      [&solution](const std::array<double, 3>& local_x) {
        return std::array<double, 1>{get(get<Pressure>(solution.variables(
            tnsr::I<double, 3>(local_x), 0., tmpl::list<Pressure>{})))};
      },
      x, d, 1e-4);
```

Reply:
Ah, thank you!